src/containers/WelcomePage/components/LastUsedContent.tsx

```diff
@@ -29,13 +41,8 @@ const LastUsedContent = ({ articleId, content, locale, t }) => {
     }
   }, []);
 
-  let resourceType = {};
-  if (
-    contexts &&
-    contexts.length > 0 &&
-    contexts[0].resourceTypes &&
-    contexts[0].resourceTypes.length > 0
-  ) {
+  let resourceType: ResourceType | any = {};
+  if (contexts[0]?.resourceTypes?.length) {
```

Comment:
Skal ikkje ? komme før [0]? Denne vil gjerne feile dersom context er undefined. Dersom det funker, så bort fra dette.
```suggestion
  if (contexts?[0].resourceTypes?.length) {
```